packages/@react-native-windows/cli/src/runWindows/utils/vsInstalls.ts

```diff
@@ -43,7 +43,9 @@ function vsWhere(args: string[], verbose?: boolean): any[] {
     );
   }
 
-  const cmdline = `"${vsWherePath}" ${args.join(' ')} -format json -utf8`;
+  const cmdline = `cmd /c chcp 65001>nul && "${vsWherePath}" ${args.join(
+    ' ',
+  )} -format json -utf8`;
   const json = JSON.parse(execSync(cmdline).toString('utf8'));
```

Comment:
actually another option - possibly more elegant - is to only pass the encoding as a param to execSync:
https://nodejs.org/api/child_process.html#child_processexecsynccommand-options

(it's part of the options object)